.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: ❗️ Issue/Bug report
+about: Report issue or bug related to the project
+title: '[ISSUE]'
+labels: 'Type/Bug'
+assignees: ''
+
+---
+
+**Describe the Issue:**
+<!-- A clear and concise description of what the bug is. If applicable, add screenshots to help explain your problem. -->
+
+**How To Reproduce:**
+<!-- Steps to reproduce the behavior. -->
+
+**Expected behavior:**
```

Comment:
Some headings are in Sentence case and some are in Title Case. Let's stick to one of them? 

General recommendation for subheadings is to use sentence case where only the first letter of the first word is capitalized.